ui/src/avatar.ts

```diff
@@ -23,5 +23,5 @@ export enum Usage {
 }
 
 // EVENTS
-export const get = (usage: Usage, id: string): Promise<EmojiAvatar> =>
+export const getAvatar = (usage: Usage, id: string): Promise<EmojiAvatar> =>
```

Comment:
Still not a fan of prefix/suffix of the module inside the module.

Reply:
I get that but I also think wildcard imports lead to ugliness like `avatar.Avatar` or `avatar.EmojiAvatar`.